### ISSUE TITLE
FeedMe Import Error and Delete Error on new Craft 3.2 upgrade

### ISSUE THREAD

**Issue by amityweb** [craftcms/cms#4450]
### Description

I used FeeMe to import content into a multisite setup using matrix blocks in Craft v 3.1. The Matrix blocks were not set to manage on a per site basis. I am importing into a section I called Pages which has a few section types called Pages, Categories and Offices.  

I upgraded to 3.2 and trying to import the FeedMe imports again, but I get the following errors on all entries in both Pages and Categories Types in Pages. They import into Offices OK. Entries in Offices never had any matrix blocks used though, whereas Pages and Categories did, so might be related to that. 

```
↳ EN Categories Feed: Attempting to duplicate an unsaved element. - Elements.php: 881. | 26/6/2019 10:25
-- | --
error | ↳ EN Categories Feed: SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '(mydb.structureelements.level - 2)' The SQL being executed was: UPDATE structureelements SETlevel=level-2, dateUpdated='2019-06-26 09:25:53' WHERE ((lft >= 60) AND (rgt <= 61)) AND (root=381) - Schema.php: 664.
```

As its unsaved, I went to an entry in question to see if I can save it. But I cant, there is no option to save it, and the update button is greyed out. Only options are to Save As New Entry or Delete Entry. 

I assumed maybe there is a mismatch somewhere between the entries from 3.1 and 3.2, and so thought I would just delete them all and re-import again, but trying to mass delete entries gives me a "Unknown Error Occurred" message. 

So I tried deleting one by one, and many deleted fine, until I got to one, which gave me this error:

```
Database Exception – yii\db\Exception
SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '(`mydb`.`structureelements`.`level` - 2)'
The SQL being executed was: UPDATE `structureelements` SET `level`=`level`-2, `dateUpdated`='2019-06-26 09:40:12' WHERE ((`lft` >= 18) AND (`rgt` <= 19)) AND (`root`=381)
Error Info: Array
(
    [0] => 22003
    [1] => 1690
    [2] => BIGINT UNSIGNED value is out of range in '(`mydb`.`structureelements`.`level` - 2)'
)
↵
Caused by: PDOException
SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '(`mydb`.`structureelements`.`level` - 2)'
in /home/mysite/public_html/vendor/yiisoft/yii2/db/Command.php at line 1290
```

I tried deleting garbage collection to remove all the entries from the DB in case it matters but still same issue.

The entries are in a structure, and the entry in question had child entries. I just deleted all the child entries and then the parent that had the above Database Exception error, and it deleted fine. 

But trying the FeedMe import still gives this error though:
`↳ EN Pages Feed: Attempting to duplicate an unsaved element. - Elements.php: 881.`

So I am a bit stuck. Managed to delete all entries to start again, but the import still fails. 

If I go to Entries -> All Entries, there are few entries in there that dont appear in their relevant section in the sites, and when I click them in the admin they 404. Its like they dont really exist even though they show in All Entries. Dont know if all this is related. 

One important point that could be related if I am the only one with this issue... on version 3.1 I needed to have the matrix blocks show their own language content within site groups. So I did try hacking the propagation import code in the file vendor/craft/cms/src/services/Elements.php to only call _propagateElement() on sites within the same group for the import feed. This did work to some extent, I had matrix blocks syncing within sites in the same group, but it did not work once I removed my own code and saved entries again. So I guess there may be a possibility something I did resulted in the error, but I dont know. 

P.S. Whilst on topic, is there a CLI command to mass delete entries? 

Thanks

### Steps to reproduce

I used FeedMe to import matrix content blocks in a site on version 3.1, then upgraded to version 3.2 and trying to re-run the feed me imports, or trying to save, or delete entries. 

### Additional info

- Craft version: Craft Solo 3.2.0-RC1
- PHP version: 7.2.10
- Database driver & version:  MySQL 5.5.60
- Plugins & versions:

Contact Form	2.2.5
Contact Form Extensions	1.1.6
Control Panel Nav	2.0.9
Craft Commerce	2.1.7
Feed Me	4.1.0
Flintec	1.0.0
GeoMate	v1.0.4
ImageOptimize	1.5.7
Queue Manager	1.1.0
Redactor	2.3.3.2
Site Switcher	2.1.0
Stripe for Craft Commerce	2.0.1.1
Super Table	2.1.20

**Comment by brandonkelly** [comment]
We just released Craft 3.2 RC2 with a fix for that “Attempting to duplicate an unsaved element” error you’re getting.

> As its unsaved, I went to an entry in question to see if I can save it. But I cant, there is no option to save it, and the update button is greyed out. Only options are to Save As New Entry or Delete Entry.

That is expected behavior in Craft 3.2 if there are no unsaved changes. Once you change something, your changes will be autosaved into a draft, at which point the “Update entry” button will become enabled.

**Comment by brandonkelly** [comment]
The SQL error you were getting should be fixed in the next Craft release – that looks to be a duplicate of craftcms/cms#3456.
